Content.Server/DeviceLinking/Systems/SignalTimerSystem.cs

```diff
@@ -129,7 +131,20 @@ private void OnTimerStartMessage(EntityUid uid, SignalTimerComponent component,
     {
         if (!IsMessageValid(uid, args))
             return;
+        OnStartTimer(uid, component);
+    }
+
+    private void OnSignalReceived(EntityUid uid, SignalTimerComponent component, ref SignalReceivedEvent args)
+    {
+        RemComp<ActiveSignalTimerComponent>(uid);
+        if (args.Port == component.Trigger)
+        {
+            OnStartTimer(uid, component);
+        }
```

Comment:
Why are you removing an active timer? It's already checked in the method so this is unnecessary.

Reply:
fix